libexec/Makefile.am

```diff
@@ -0,0 +1,25 @@
+# FIXME:
+#
+# 1. SCRIPTS makes all this stuff executable when it shouldn't be, but
+#    Automake complains that DATA can't go in libexecdir.
```

Comment:
I wonder if it would complain if we try to put executable scripts in `lib`. The reason I initially proposed libexec was because `test` and `examples` had a mixture of data and executables intended to be used by other programs, e.g., `make test`, `make-auto`, .etc.

Reply:
Legit. Let's continue that discussion in #596.